matrix/utils.py

```diff
@@ -227,6 +227,8 @@ def translate_ansi_colors(entity):
 
     '''
     log.info("Running crash dump")
+    if controller:
+        model_name = "{}:{}".format(controller, model_name)
```

Comment:
Sounds good.  Let's leave it the way it is now and make it future Pete's problem.

Or.. ya know..